msdk-featdet/msdk-featdet-openms/src/main/java/io/github/msdk/featdet/openms/FeatureFinderMetaboMZValueDetector.java

```diff
@@ -0,0 +1,83 @@
+package io.github.msdk.featdet.openms;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.Nonnull;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Node;
+import org.w3c.dom.Element;
+import org.xml.sax.SAXException;
+
+import io.github.msdk.datamodel.rawdata.RawDataFile;
+
+public class FeatureFinderMetaboMZValueDetector {
+	private final String OPENMS_FEATURE_FINDER_METABO_LIBRARY_NAME = "FeatureFinderMetabo";
+
+	private final @Nonnull File mzMLFile;
+	private final @Nonnull File featureXMLFile;
+
+	public FeatureFinderMetaboMZValueDetector(String mzMLFileName, String featureXMLFileName) {
```

Comment:
In fact, since we are in MSDK, we don't really need the output featureXMLFileName. It is better to redirect the OpenMS output into a temporary file, then parse the contents, and delete the temporary file afterwards. So you can remove the featureXMLFileName parameter.